js/views/file_input_view.js

```diff
@@ -281,6 +282,19 @@
             e.stopPropagation();
             e.preventDefault();
             this.$el.removeClass('dropoff');
+        },
+        onPaste: function(e) {
+            var items = e.originalEvent.clipboardData.items;
+            var imgBlob = null;
+            for (var i = 0; i < items.length; i++) {
+                if (items[i].type.indexOf("image") == 0) {
```

Comment:
The linter wants `===` for comparison with 0. Also, a stricter parsing: `items[i].type.split('/')[0] === 'image'` would reject oddball content types like `'imageish/hack'`.